test/junit/scala/sys/process/ProcessTest.scala

```diff
@@ -102,14 +103,23 @@ class ProcessTest {
     val out = createTempFile("out", "tmp")
     val outf = out.toFile
 
-    try {
-      val p0 = (file1.toFile : ProcessBuilder.Source).cat #&& pb2
-      val p = p0 #> outf
+    val stackless = stacklessly {
+      try {
+        val p0 = (noFile.toFile : ProcessBuilder.Source).cat #&& pb2
+        val p = p0 #> outf
 
-      assertEquals(1, p.!)
-      assertFalse(failed.get)
-    } finally {
-      Files.delete(out)
+        assertEquals(1, p.!)
+        assertFalse(failed.get)
+      } finally {
+        Files.delete(out)
+      }
+    }
+    stackless.run()
+    stackless.assert()
```

Comment:
I'd have made this part of `.run()`, but this is more separation-of-concerns-y.

Reply:
Yeah if the intended usage is to run and assert, I'd make `run` assert, and have a more verbose `runWithoutAssert` path-least-taken option.  (But not a blocker..)